NEWS.md

```diff
@@ -41,11 +41,13 @@ Standard library changes
 
 * `dot` now admits a 3-argument method `dot(x, A, y)` to compute generalized dot products `dot(x, A*y)`, but without computing and storing the intermediate result `A*y` ([#32739]).
 
-#### SparseArrays
+#### Random
 
+* `AbstractRNG`s now behave like scalars when used in broadcasting ([#33213]).
```

Comment:
```suggestion
* `AbstractRNG`s now behave like scalars when used in broadcasting ([#31071]).
```
I think convention is for the link to be to the issue that is closed,
and only to the PR if there is no issue.

I am not sure though, so wait for somone else to chime in

Reply:
I do not know - and would like to be advised

But if it should link to the issue (not the PR) we should update https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#news-worthy-changes

Reply:
Why not both?

Reply:
```suggestion
* `AbstractRNG`s now behave like scalars when used in broadcasting ([#31071, #33213]).
```

Reply:
We normally link just the PR, I think.

Reply:
changed back

Reply:
My bad. Good to know